packages/cli/src/deploy/configToModules.ts

```diff
@@ -14,8 +14,18 @@ export async function configToModules<config extends World>(
   // TODO: remove/replace `forgeOutDir`
   forgeOutDir: string,
 ): Promise<readonly Module[]> {
+  const configModules = [
+    {
+      name: "metadata",
+      artifactPath: "@latticexyz/world-modules/out/MetadataModule.sol/MetadataModule.json",
```

Comment:
need to import this way otherwise we create a circular dep between CLI and world-modules

downside is this would require projects to have a world-modules dep in their package that uses `mud` CLI

maybe could make this more explicit as a peer dep for CLI?